test/gpu/native/studies/chop/chplGPU.skipif

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Skip if not doing performance testing
+if [ -z "$CHPL_TEST_PERF" ]; then
+  echo "True"
+  exit
+fi
+
+# Skip this test if we cannot connect to github, which is required to
+# run the test.
+ping -c 1 github.com &> /dev/null
+status=$?
+if ! [ $status -eq 0 ]; then
+  echo "True"
+fi
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+#CHOP_URL=https://github.com/tcarneirop/ChOp.git
+CHOP_URL=git@github.com:tcarneirop/ChOp.git
+CHOP_BRANCH=main
```

Comment:
I think we should allow url/branch to take defaults:

```
CHOP_URL =${CHOP_URL:-git@github.com:tcarneirop/ChOp.git}
CHOP_BRANCH =${CHOP_BRANCH:-main}
```

Reply:
I'll use "https://github.com/tcarneirop/ChOp.git" as the default but otherwise take this suggestion directly.